file_app/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+# ======================================= FileData Model ============================================
+class FileData(models.Model):
+    # file_name = models.ForeignKey(FileName,on_delete=models.CASCADE,default="")
+    file_name = models.CharField(max_length=200,default="")
+    max_temp = models.IntegerField(default=0)
+    min_temp = models.IntegerField(default=0)
+    max_humd = models.IntegerField(default=0)
+    min_humd = models.IntegerField(default=0)
+    mean_humd = models.IntegerField(default=0)
+    # avg_max_temp = models.IntegerField(default=0)
```

Comment:
we should store all data from files to db, for our record and in APIs we'll handle only that data which is needed for that task. but all data of files should be imported to db